samples/apps/copilot-chat-app/README.md

```diff
@@ -14,9 +14,9 @@ functions that work together to construct each response.
 
 # Configure your environment
 Before you get started, make sure you have the following requirements in place:
-- [.NET 6.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)
-- [Node.js](https://nodejs.org/en/download)
-- [Yarn](https://classic.yarnpkg.com/lang/en/docs/install)
+- [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+- [Node.js](https://nodejs.org/)
```

Comment:
the links for node and Yarn don't go to the install/download pages. Was this intentional?

Reply:
Yes, exploring the web site they had different structures in other languages, so it seemed best to provide a link to the root page and allow users to review and confirm the materials they are downloading.